src/main/cljs/dmohs/react/core.cljs

```diff
@@ -152,7 +152,10 @@
           bound-method))))
 
 (defn after-update [instance f & args]
-  (.setState instance #js{} (fn [] (apply f args))))
+  (.forceUpdate instance (fn [] (apply f args))))
+
+(defn force-update [instance]
```

Comment:
I'd be inclined to deprecate `after-update` and give `force-update` multiple arities just like React's.

Reply:
How do you want me to deprecate it? Throw an error? Just map it under the hood? Part of the issue here is that after-update, when it was using setState, was guaranteed to be batched with the state update that it wanted to be after; I don't know for a fact that forceUpdate will be batched. An alternative is to have an update-state-with-callback function that can be destructured that gets the new state (i.e. `(@state assoc :foo "bar")`) and a callback as the second argument, and then have force-update for external data situations only, as was intended...